.github/workflows/main.yml

```diff
@@ -320,3 +320,35 @@ jobs:
           path: |
             **/junit-*.xml
             **/TEST-*.xml
+
+  jboss:
```

Comment:
[minor] we could make that step a sub-step of the application server tests,but as it's quite fast (about 6min) in CI that's fine for now and allows us to keep an eye on it for now in case of failure.